src/configuration/copy.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Thoughts on naming this file something like `authorityCopy` or something else more specific?